src/WebJobs.Extensions.DurableTask/ManagedIdentityTokenSource.cs

```diff
@@ -32,6 +32,11 @@ public ManagedIdentityTokenSource(string resource, ManagedIdentityOptions option
             {
                 this.Resource = "https://management.core.windows.net/.default";
             }
+
+            if (this.Resource.Equals("https://graph.microsoft.com") || this.Resource.Equals("https://graph.microsoft.com/"))
```

Comment:
nit: maybe make this an `else if` instead of an `else` to avoid unnecessary string comparisons.

Reply:
Do you mean make the `if` that checks the graph URL an `else if`? Just want to confirm before I make any changes.

Reply:
Yep!